parts/linux/cloud-init/artifacts/ci-syslog-watcher.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Update CI syslog config based on status change
+
+[Service]
+Type=oneshot
+ExecStart=/usr/local/bin/ci-syslog-watcher.sh
```

Comment:
nit: do you care about Restart or RestartSec? default I think is no restart so would trigger on next path change, if you do Restart=on-failure you probably want to increase RestartSec to avoid spamming it? minor stuff tho

Reply:
if it fails it fails and we wait for the next path change to trigger it